packages/flash/src/index.ts

```diff
@@ -18,8 +19,11 @@ export function flashMiddleware() {
 }
 
 export function flash(type?: FlashMessageType | FlashMessageType[]): FlashMessage[];
-export function flash(type: FlashMessageType, message: string): void;
-export function flash(type?: FlashMessageType | FlashMessageType[], message?: string) {
+export function flash(type: FlashMessageType, message: string | HtmlSafeString): void;
```

Comment:
The message could also be declared as type `HtmlValue`, which would allow other types of message to be accepted, but I'm not convinced this would be useful.

Reply:
`HtmlSafeString` makes sense here 👍